src/test/scala/org/broadinstitute/dsde/workbench/sam/service/MockUserServiceBuilder.scala

```diff
@@ -137,6 +145,9 @@ case class MockUserServiceBuilder() extends IdiomaticMockito {
   }
 
   private def makeUsers(samUsers: Iterable[SamUser], mockUserService: UserService): Unit = {
+    mockUserService.getUserFromGoogleSubjectId(any[GoogleSubjectId], any[SamRequestContext]) answers ((googleSubjectId: GoogleSubjectId) =>
```

Comment:
This is necessary to get the StandardSamUserDirectives.getActiveSamUser and StandardSamUserDirectives.getSamUser calls to work